torch/_export/serde/schema.py

```diff
@@ -8,7 +8,7 @@
 from torch._export.serde.union import _Union
 
 # NOTE: Please update this value if any modifications are made to the schema
-SCHEMA_VERSION = (4, 2)
+SCHEMA_VERSION = (5, 1)
```

Comment:
Why does it go down from 2 to 1? Should we leave a comment explain what these numbers are for?

Reply:
here is a doc https://docs.google.com/document/d/1EN7JrHbOPDhbpLDtiYG4_BPUs7PttpXlbZ27FuwKhxg/edit